packages/lesswrong/components/common/Header.tsx

```diff
@@ -26,6 +30,62 @@ export const HEADER_HEIGHT = isBookUI ? 64 : 66;
 /** Height of top header on mobile. On Friendly UI sites, this is the same as the HEADER_HEIGHT */
 export const MOBILE_HEADER_HEIGHT = isBookUI ? 56 : HEADER_HEIGHT;
 
+const textColorOverrideStyles = (
```

Comment:
Factored out so we can easily create overrides with different colors instead of just white